### DISCUSSION TITLE
GitHub Action with maturin worked 20 days ago but failed today on Linux

### DISCUSSION THREAD

**Discussion by CarlKCarlK** [PyO3/maturin#724] [Q&A]
I didn't change anything, but my GitHub Action started failing yesterday with 
> \> maturin develop
[log file](https://github.com/fastlmm/bed-reader/runs/4383601419?check_suite_focus=true):

```
💥 maturin failed
  Caused by: failed to create file `/home/runner/work/bed-reader/bed-reader/target/wheels/bed_reader-0.2.7-cp37-cp37m-linux_x86_64.whl`
  Caused by: Permission denied (os error 13)
Error: Process completed with exit code 1.
```
It is only failing on Linux. It still works on MacOS.

I'd welcome ideas on how to debug what's going on.

Thanks!
p.s. This is for bed-reader, a Python/Rust library for reading genomics files.

**Comment by messense** [comment]
It might be related to docker, see https://vsupalov.com/docker-shared-permissions/

Edit: transferred this discussion from pyo3 to maturin.
